docs/build/build-options.md

```diff
@@ -284,6 +284,9 @@ When ON, enables logging in the *haicrypt* module, which serves as a connector t
 an encryption library. Logging here might be seen as unsafe, therefore this 
 option is turned OFF by default.
 
+The log corresponds to the `HAICRYPT` functionl area.
+For example, to enable it in `srt-live-transmit` use `--logfa haicrypt`
+with combination of the required `--loglevel`.
```

Comment:
I'd write more explicit:

Note that this only enables haicrypt logging in the build, but it doesn't enable haicrypt functional area in the logging configuration. This must be requested as usual by enabling the haicrypt functional area (in case of applications, for example, you should use `--logfa haicrypt`).

Note also that this describes a build option, the FA should be enabled in the log configuration (API: `srt_addlogfa`) and you are describing a command-line option for `srt-live-transmit`.

Reply:
Updated